helix-term/src/commands.rs

```diff
@@ -1988,6 +2007,11 @@ fn global_search(cx: &mut Context) {
                 let picker = FilePicker::new(
                     all_matches,
                     current_path,
+                    if editor.config().icons.picker {
+                        Some(&editor.icons)
+                    } else {
+                        None
+                    },
```

Comment:
```suggestion
                    editor.config().icons.picker.then(|| &editor.icons),
```

The pattern comes up again, I've marked them all (1) to keep track.